Add announcements for week 9, 2021 (#635)
giantswarm/releases: announcements/2021/week09.md

# :zap: This week in provider-independent announcements (March 5, 2021) :zap:

For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.

## Managed apps

> [NGINX IC v1.15.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.15.0/) upgrades the container images (controller container to v0.44.0 and kube-webhook-certgen to v1.5.1). It also removes conflicting admission webhook api versions...

Reviewer: @QuentinBisson i think you removed conflicting admission  webhook, etc. for a specific reason? Maybe add one line here on the value of that? cc @giantswarm/app-squad-nginx

Reviewer 2: https://github.com/giantswarm/nginx-ingress-controller-app/pull/178#issuecomment-784179339 explains it a bit more

```
Sure, so basically, the default validation webhook matching mode is Equivalent by default since webhook v1. This policy will ensure that the tree of apiversions will be valid according the the webhook.
```

Reviewer 3: Nice one

> [NGINX IC v1.15.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.15.0/) upgrades the container images (controller container to v0.44.0 and kube-webhook-certgen to v1.5.1). It also removes conflicting admission webhook api versions...

Reviewer: ```suggestion
[NGINX IC v1.15.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.15.0/) upgrades the container images (controller container to v0.44.0 and kube-webhook-certgen to v1.5.1). It also removes conflicting admission webhook api versions, ensuring the the tree of apiversions will be valid according to the webhook.
```

TY @ubergesundheit and @QuentinBisson Does this sound good to you?

> [NGINX IC v1.15.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.15.0/) upgrades the container images (controller container to v0.44.0 and kube-webhook-certgen to v1.5.1). It also removes conflicting admission webhook api versions, ensuring the the tree of apiversions will be valid according to the webhook.

Reviewer: @QuentinBisson looks good to you? Seems Gerald is AFK haha

Reviewer 3: Yes this works for me ;)

Reviewer: thanks!

> [NGINX IC v1.15.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.15.0/) upgrades the container images (controller container to v0.44.0 and kube-webhook-certgen to v1.5.1). It also removes conflicting admission webhook api versions, ensuring the the tree of apiversions will be valid according to the webhook.

Reviewer: ```suggestion
[NGINX IC v1.15.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.15.0/) upgrades the container images (controller container to v0.44.0 and kube-webhook-certgen to v1.5.1). It also removes conflicting admission webhook api versions, ensuring the tree of apiversions will be valid according to the webhook.
```

> [Kong v1.2.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.2.0/) upgrades Kong to v2.3...

Reviewer: @giantswarm/app-squad-kong can you help here? there's a note that says 'important' but then I don't know what's 1.15.0, etc... what's important for customers to know here? https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.2.0/

Also, `upgrade documentation` goes to 404... https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.2.0/helm/kong-app/UPGRADE.md

> [Kong v1.2.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.2.0/) upgrades Kong to v2.3...

Reviewer: ```suggestion
[Kong v1.2.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.2.0/) upgrades to upstream chart v1.15.0 and Kong to v2.3. Upstream chart v1.15.0 is an interim release before the planned v2.0.0 release. Please review the [upstream changelog](https://github.com/giantswarm/kong-app/blob/master/helm/kong-app/CHANGELOG.md) for all changes between v1.12.0 and v1.15.0.
```

[Loki v0.1.1-alpha](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.1.1-alpha/) adds annotation to route alerts to the apps team, in preparation for offering Loki as a managed app in production.

## Apps supported with best-effort

[Jaeger Operator v0.2.3](https://docs.giantswarm.io/changes/playground-apps/jaeger-operator-app/v0.2.3/) upgrades to upstream v0.2.3.

> [Fluent Logshipping v0.6.1](https://docs.giantswarm.io/changes/playground-apps/fluent-logshipping-app/v0.6.1/) fixes the output config format for AWS outputs, as well as mounts journald path and sets in fluent-big config.

Reviewer: @yasn77 check if you are fine with this 🙏 ?

> [Fluent Logshipping v0.6.1](https://docs.giantswarm.io/changes/playground-apps/fluent-logshipping-app/v0.6.1/) fixes the output config format for AWS outputs, as well as mounts journald path and sets in fluent-big config.

Reviewer 4: ```suggestion
[Fluent Logshipping v0.6.1](https://docs.giantswarm.io/changes/playground-apps/fluent-logshipping-app/v0.6.1/) fixes the output config format for AWS outputs, as well as mounts journald path and sets it correctly in fluent-bit config.
```

> [Jaeger Operator v0.2.3](https://docs.giantswarm.io/changes/playground-apps/jaeger-operator-app/v0.2.3/) upgrades to upstream v0.2.3.

Reviewer: @ced0ps what is the right version? there's nothing in changelog 🙈  and in the PR it says v2.19.1 https://github.com/giantswarm/jaeger-operator-app/pull/30/files

but in v0.2.2 https://github.com/giantswarm/jaeger-operator-app/blob/master/CHANGELOG.md#022---2021-01-20, chart is 1.21.2 and jaeger-all-in-one is 1.21.0? What is 2.19.1? 🙏

> [Strimzi Kafka Operator v0.2.0](https://docs.giantswarm.io/changes/playground-apps/strimzi-kafka-operator-app/v0.2.0/) upgrades to upstream v0.21.1.

Reviewer: @yasn77 check if you are fine with this 🙏 ?

Reviewer 4: That's cool

Reviewer: thanks!!!
